<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectAxisBarClass</class>
 <widget class="QWidget" name="SelectAxisBarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#E6E6E6;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>73</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="ltp::base::ClickDelayCheckButton" name="axisButton1_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8;color:white;}
QPushButton:hover{background:#0064C8;color:white;}


</string>
       </property>
       <property name="text">
        <string>X</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ltp::base::ClickDelayCheckButton" name="axisButton2_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8; color:white;}
QPushButton:hover{background:#0064C8; color:white;}


</string>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ltp::base::ClickDelayCheckButton" name="axisButton3_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8; color:white;}
QPushButton:hover{background:#0064C8; color:white;}


</string>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ltp::base::ClickDelayCheckButton" name="axisButton4_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8; color:white;}
QPushButton:hover{background:#0064C8; color:white;}</string>
       </property>
       <property name="text">
        <string>A</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ltp::base::ClickDelayCheckButton" name="axisButton5_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>4</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8; color:white;}
QPushButton:hover{background:#0064C8; color:white;}</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="axisButton6_">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background:transparent;border:0px;font:22px Arial;}
QPushButton:checked{background:#0064C8; color:white;}
QPushButton:hover{background:#0064C8; color:white;}</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>64</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12px Arial Bold;color:#0064C8</string>
     </property>
     <property name="text">
      <string>手轮</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font:12px Arial Bold;color:#0064C8</string>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ltp::base::ClickDelayCheckButton</class>
   <extends>QPushButton</extends>
   <header>base/clickdelaycheckbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
